e2e-tests/denylist/run-tests.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+source ../util.sh
+
+echo "------------------------------------"
+echo "Cleaning up running Docker containers and volumes, and key material..."
+echo "------------------------------------"
+docker compose down
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to shut down old containers"
+	exitWithDockerLogs 1
+fi
+
+docker compose rm -f -v
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to remove old containers"
+	exitWithDockerLogs 1
+fi
+
+echo "------------------------------------"
+echo "Starting Docker containers..."
+echo "------------------------------------"
+docker compose up --wait
+if [ $? -ne 0 ]; then
+	echo "ERROR: failed to start containers"
+	exitWithDockerLogs 1
+fi
+
+sleep 5
+
+# Simply to log this
+openssl version
+
+echo "------------------------------------"
+echo "Connecting (defaults) with allowed cert.."
+echo "------------------------------------"
+for x in $(seq 100); do
+	# Connect to the nuts-node with a valid client certificate
+	#curl --fail --cert client-allowed.crt --key client-allowed.key "https://localhost:1323/status/diagnostics"
+	openssl s_client -connect localhost:15555 -cert client-allowed.crt -key client-allowed.key -CAfile truststore-development.pem -verify_return_error -tls1_2 < <(echo "Hello Nuts 🥜")
+	if [ $? -ne 0 ]; then
+		echo "ERROR: failed to contact nuts-node-defaults with valid certificate"
+		exitWithDockerLogs 1
+	fi
+
+	# Check the logs have the right contents
+	docker logs denylist-nuts-node-defaults-1 2>&1 | tail -n1 | grep 'Validated certificate'
```

Comment:
adding `container_name: <some-name>` to a service in the docker-compose file sets a reliable name to use instead of `denylist-nuts-node-defaults-1`